osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -121,6 +120,16 @@ private void ensureLoaded(OsuGameBase osu, int timeout = 100)
 
             Assert.IsTrue(beatmap.HitObjects.Count > 0);
         }
+
+        /// <summary>
+        /// Creates a copy of the original .osz file
+        /// </summary>
+        /// <returns>Path to the created copy</returns>
+        private string prepareOszFileForTest()
+        {
+            var original = new FileInfo(osz_path);
+            return original.CopyTo(Path.GetFileName(original.FullName), true).FullName;

```

Comment:
Current working directory(running .exe file location). I did not think about it too much since this file will be removed after test run.